main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,260 @@
+# ERTP Guide
+
+[ERTP](https://agoric.com/documentation/glossary/#ertp) (*Electronic
+Rights Transfer Protocol*)
+is Agoric's token standard for transferring tokens and other digital assets in
+JavaScript. Using the
+[ERTP API](https://agoric.com/documentation/ertp/api/),
+you can easily create and use digital assets, all of which are
+transferred exactly the same way and with exactly the same security properties. 
+
+ERTP uses
+*[object capabilities](https://agoric.com/documentation/glossary/#object-capabilities)*
+to enforce access control. If your program has a reference to an
+object, it can call methods on that object. If it doesn't have a
+reference, it can't. For more on object capabilities, see
+[Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
+
+## Objects Overview
+
+![ERTP object relationships](./assets/relationships.svg)  
```

Comment:
I think this diagram will be more confusing than helpful, but the idea of having a diagram is very useful. I think this particular diagram doesn't mention digital assets and doesn't have enough of a hierarchy - it seems like everything is at the same level. Can we switch to something like this:

![IMG_20200731_151022](https://user-images.githubusercontent.com/2441069/89081753-e285ab00-d340-11ea-8b15-f9af698d1e35.jpg)

Brand shouldn't be its own section in the diagram, but should be describing everything else. Digital assets should be in the center. Amounts should be off of digital assets. We may want to explain more of what the issuer does in detail in a separate diagram. I don't have very good ideas for that, but here's a representation of what is stored in the issuer.
![IMG_20200731_150858](https://user-images.githubusercontent.com/2441069/89081839-21b3fc00-d341-11ea-9419-87415779cb80.jpg)